crates/sui-types/src/transaction.rs

```diff
@@ -3361,3 +3361,9 @@ impl TransactionKey {
         }
     }
 }
+
+impl From<TransactionDigest> for TransactionKey {
```

Comment:
I think I deliberately didn't implement this because I was worried it might lead to people erroneously calling `tx.digest().into()` instead of `tx.key()` on a `Transaction`. I don't strongly oppose this if you think that's not a big concern, though

Reply:
oh thats a pretty good point. will remove